cfgov/ask_cfpb/wagtail_hooks.py

```diff
@@ -97,3 +98,44 @@ def register_export_menu_item():
 @hooks.register('register_admin_urls')
 def register_export_url():
     return [url('export-ask', export_data, name='export-ask')]
+
+
+@hooks.register('after_create_page')
+def create_answer_id(request, page):
+    """
+    Create an incremented Answer ID for a new AnswerPage and attach it.
+
+    Also create a sister-language page to keep languages in sync.
+    """
+    if isinstance(page, AnswerPage) and page.answer_base is None:
+        new_answer_base = Answer(
+            last_user=request.user,
+            question=page.title)
+        new_answer_base.save()
+        new_id = new_answer_base.pk
+        if page.get_parent().slug == 'obtener-respuestas':
```

Comment:
This is minor but it might be nice to make this a function that takes in the parent slug ('obtener-respuestas' or 'ask-cfpb') as the argument and then returns the sister page. Then page.language and the slug could be passed in as variables instead of hardcoded.